docker/check-snapshot/check.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+
+from data_repo_client import ApiClient, Configuration, RepositoryApi
+import google.auth
+from google.auth.transport.requests import Request
+from google.cloud import bigquery
+
+
+logging.basicConfig(level=logging.INFO)
+
+ENV = os.getenv("ENV")
+google_project = os.getenv("REPO_DATA_PROJECT")
+data_repo_host = os.getenv("REPO_HOST")
+dataset_name = os.getenv("REPO_DATASET_NAME")
+dataset_id = os.getenv("REPO_DATASET_ID")
+
+
+def default_google_access_token():
```

Comment:
🌱 I am seeing a common pattern of jade + bq scripts that work off of default google app tokens and the repo client. 🤔 what if we create a shared lib for such things? (out of scope for this PR obviously)

Reply:
I've thought about this too; I hope/imagine the need would be reduced once we simply write this code in Python for Dagster, so I didn't want to invest in code for the Argo Workflow way of doing things. If we'd still find it useful to have a library for these things though then yeah we should!